flexmeasures/utils/tests/test_coding_utils.py

```diff
@@ -0,0 +1,24 @@
+from flexmeasures.utils.coding_utils import deprecated
+import warnings
+
+
+def test_deprecated_decorator():
+
+    # defining a function that is deprecated
+    @deprecated("other place for the function")
+    def deprecated_function():
+        pass
+
+    with warnings.catch_warnings(record=True) as w:
+        # Cause all warnings to always be triggered.
+        warnings.simplefilter("always")
+
+        deprecated_function()  # calling a deprecated function
+
+        assert len(w) == 1  # only 1 warning being printed
+        assert issubclass(
+            w[-1].category, FutureWarning
+        )  # warning type is DeprecationWarning
```

Comment:
Looks like this comment hadn't been updated?

Reply:
Yes, I'm updating it